### DISCUSSION TITLE
Práctica CSS casi lista

### DISCUSSION THREAD

**Discussion by khisus19** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#351] [Show and tell]
Acabo de terminar el footer. Traté de hacerla lo más parecida posible. Solo me faltaron los iconos. 

No la hice responsive porque la semana pasada me sentí muy presionado para entregar mi práctica a tiempo y lo hice a última hora por tratar de hacerle de todo. Esta vez mi meta fue hacerla lo más semejante que pudiera sin preocuparme por lo responsive. Tampoco le puse los triángulos que hay en el background.

https://khisus19.github.io/vaccination-landing-page/

Les dejo el enlace al repo con todos los recursos que usé por si quieren usar las imágenes o cualquier otra cosa.
Link del repo: https://github.com/khisus19/vaccination-landing-page.git

Y ya que estamos les dejo el enlace a la página de la práctica anterior que no la había compartido. Esta si es responsive.
https://khisus19.github.io/delicias-del-cielo/

**Comment by AlejandroMontoyaT** [comment]
Gracias master. yo e tenido problemos con los div y los Grid basicamente en todo el css.

**Reply by ArielCanedo** [reply]
Escribe tus dudas bro
Aquí las vamos respondiendo o en otro hilo

**Comment by OscarPedroza** [comment]
Muy buen trabajo compañero!
